src/app/app.module.js

```diff
@@ -0,0 +1,37 @@
+ 'use strict';
+
+ angular.module('esn-frontend-account-inbox', [
+        'op.dynamicDirective',
+        'esn.session',
+        'esn.configuration',
+        'esn.async-action',
+        'esn.i18n',
+        'esn.subheader',
+        'esn.inbox.libs'
+    ]);
+
+require('esn-frontend-inbox/src/esn.inbox.libs/app/app.module.js')
+require('esn-frontend-inbox/src/esn.inbox.libs/app/app.constants.js');
+require('esn-frontend-inbox/src/esn.inbox.libs/app/services/inbox-restangular.service.js');
+require('esn-frontend-common-libs/src/frontend/components/dynamic-directive/src/dynamic-directive.js');
```

Comment:
This one also: Please remove this since you do not use dynamic directive anymore